app/test_main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+# git commit -am 'Solution'
+from unittest import mock
+from app.main import can_access_google_page
```

Comment:
you should split this imports with empty line

Reply:
Fixed

Reply:
What about my task?

Reply:
What about my task?